pages/express-relay/integrate-as-searcher/svm.mdx

```diff
@@ -125,32 +139,41 @@ import * as limo from "@kamino-finance/limo-sdk";
   * @returns The generated bid object
   */
 async generateBid(opportunity: OpportunitySvm): Promise<BidSvm> {
```

Comment:
thanks for updating these code blocks. can you just make sure the docstring describing the functions are up-to-date as well?

Reply:
Sure